cli.js

```diff
@@ -4,19 +4,26 @@ const open = require('open')
 const readline = require('readline')
 const dedent = require('dedent')
 const getRepo = require('./lib/get-repo')
+const getCurrentRepo = require('./lib/get-current-repo')
 
 if (process.stdin.isTTY) {
   // Read from arguments
   const names = process.argv.slice(2)
   if (!names.length) {
-    console.log(dedent`To use ghub, specify one or many package names.
+    // Get current repo, if any
```

Comment:
The code is simple and clear enough here that the comments don't seem necessary.